<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lancement_combat</class>
 <widget class="QDialog" name="Lancement_combat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>582</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>582</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>582</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="Ensemble_lancement">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>951</width>
     <height>591</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCommandLinkButton" name="PASS">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Images_interface/Pass.webp</normaloff>../Images_interface/Pass.webp</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="VS">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>961</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images_interface/VS.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Joueur">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>351</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Nom_Pokemon_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Pokemon_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Nom_Pokemon_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Pokemon_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Pokemon_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Nom_Pokemon_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Nom_adversaire">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>450</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Adversaire">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Pokemon_images/Abra.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Type_1">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>370</y>
      <width>47</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Type_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>370</y>
      <width>47</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>VS</zorder>
   <zorder>PASS</zorder>
   <zorder>Joueur</zorder>
   <zorder>Nom_Pokemon_3</zorder>
   <zorder>Pokemon_3</zorder>
   <zorder>Nom_Pokemon_2</zorder>
   <zorder>Pokemon_1</zorder>
   <zorder>Pokemon_2</zorder>
   <zorder>Nom_Pokemon_1</zorder>
   <zorder>Nom_adversaire</zorder>
   <zorder>Adversaire</zorder>
   <zorder>Type_1</zorder>
   <zorder>Type_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
